extension/scripts/features/casino-slots-animation-speed/ttSlotsAnimationSpeed.entry.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+(() => {
+    featureManager.registerFeature(
+        "Fast Slots Animation", 
+        "casino", 
+        isEnabled,
+        init, 
+        start, 
+        cleanup, 
+        {
+			storage: ["settings.pages.casino.fastSlotsAnimations"],
+		}, 
+        requirements, 
+        {}
+    );
+
+    function isEnabled() {
+        return !!settings.pages.casino.fastSlotsAnimations;
+    }
+
+    function init() {
+        $.ajaxSetup({
+            dataFilter: ajaxFilter
+        });
+    }
+
+    function start() {
+
+    }
+
+    function cleanup() {
+
+    }
+
+    function requirements() {
+        return null;
+    }
+
+    function ajaxFilter(data) {
+        if (!data) return data;
+        if (!isEnabled()) return data;
+        
+        try {
+            var tempData = JSON.parse(data);
+            if (tempData?.barrelsAnimationSpeed) {
+                tempData.barrelsAnimationSpeed = parseInt(tempData.barrelsAnimationSpeed / 10);
```

Comment:
I've noticed that doing it this way in your script often results in a 'slots already running ' or similar. = 100 seems to being better results.

Reply:
Really! I don't see that and I'm here tapping the button non-stop, unlike with the other script floating out there. I'll give that a test, maybe I just put in a min value instead of mutating the value with a scaling factor to make sure it's long enough to not get the warning message from Torn.